testbed/testbed/receivers.go

```diff
@@ -26,12 +26,12 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector/receiver/opencensusreceiver"
 )
 
-// Receiver allows to receive traces or metrics. This is an interface that must
+// DataReceiver allows to receive traces or metrics. This is an interface that must
 // be implemented by all protocols that want to be used in MockBackend.
-// Note the terminology: testbed.Receiver is something that can listen and receive data
+// Note the terminology: testbed.DataReceiver is something that can listen and receive data
 // from Collector and the corresponding entity in the Collector that sends this data is
-// an exporter.
-type Receiver interface {
+// an sender.
```

Comment:
Here it will be an exporter :)

Reply:
Thanks for catching. Too many search/replaces in comments. :-)